packages/sit-onyx/src/components/OnyxDataGrid/features/sorting/sorting.ts

```diff
@@ -0,0 +1,68 @@
+import { computed, h, ref } from "vue";
+import { createFeature } from "..";
+import { injectI18n } from "../../../../i18n";
+import type { DataGridEntry } from "../../types";
+import SortAction from "./SortAction.vue";
+
+export type SortDirection = "asc" | "desc" | "none";
+
+export const nextSortDirection = (current?: SortDirection): SortDirection => {
+  switch (current) {
+    case "asc":
+      return "desc";
+    case "desc":
+      return "none";
+    case "none":
+    default:
+      return "asc";
+  }
+};
+
+const SORTING_FEATURE = Symbol("Sorting");
+
+export const useTableSorting = createFeature(<TEntry extends DataGridEntry>() => {
```

Comment:
```suggestion
export const useDataGridSorting = createFeature(<TEntry extends DataGridEntry>() => {
```

Its about the data grid, not the table right?

Reply:
done